stewart/theme.json

```diff
@@ -105,6 +105,9 @@
 			"margin": {
 				"horizontal": "30px",
 				"vertical": "30px"
+			},
+			"spacing": {
+				"outer": "min(1em, 30px)"
```

Comment:
TIL

Reply:
Does this actually do anything?

Reply:
Ha, whoops. No, I meant to put a `vw` unit there instead of the `1em`. This should scale more nicely now.

Reply:
Oh! That's the variable used in all the new layout CSS here: 

https://github.com/Automattic/themes/blob/fafc2c5119fabffa2cddf78ce7112f65a1e884c9/stewart/style.css#L141-L142

It defines the outer left and right padding for the site. It's the same variable name [used in Twenty Twenty-Two](https://github.com/WordPress/twentytwentytwo/blob/0411debe4f66661d878c171d98614d8cfd0d4a5e/theme.json#L144) for easy copy/paste if this code needs to update.

Reply:
Ah nice one.